lib/core-integration/src/Test/Integration/Scenario/CLI/Transactions.hs

```diff
@@ -905,6 +915,67 @@ spec = do
               oJson1 <- expectValidJSON (Proxy @([ApiTransaction t])) o1
               oJson2 <- expectValidJSON (Proxy @([ApiTransaction t])) o2
               length <$> [oJson1, oJson2] `shouldSatisfy` all (== 0)
+
+    it "TRANS_DELETE_01 - Can forget pending transaction via CLI" $ \ctx -> do
+        wSrc <- fixtureWallet ctx
+        wDest <- emptyWallet ctx
+        addr:_ <- listAddresses ctx wDest
+        let addrStr =
+                encodeAddress (Proxy @t) (getApiT $ fst $ addr ^. #id)
+        let amt = 14
+        let (feeMin, feeMax) = ctx ^. feeEstimator $ TxDescription
+                { nInputs = 1
+                , nOutputs = 1
+                }
+        let args = T.unpack <$>
+                [ wSrc ^. walletId
+                , "--payment", T.pack (show amt) <> "@" <> addrStr
+                ]
+
+        -- post transaction
+        (c, out, err) <- postTransactionViaCLI @t ctx "cardano-wallet" args
+        err `shouldBe` "Please enter your passphrase: **************\nOk.\n"
+        txJson <- expectValidJSON (Proxy @(ApiTransaction t)) out
+        verify txJson
+            [ expectCliFieldBetween amount (feeMin + amt, feeMax + amt)
```

Comment:
Do these assertions need to be present?
I thought it should only set up the pending transaction, remove it, then verify that the transaction has been forgotten.